assets/operator/okd-x86_64/nginx/imagestreams/nginx-centos.json

```diff
@@ -92,7 +92,7 @@
 				},
 				"from": {
 					"kind": "DockerImage",
-					"name": "docker.io/centos/nginx-116-centos7:latest"
+					"name": "docker.io/centos/nginx-116-centos8:latest"
```

Comment:
This does not match what is up in https://github.com/sclorg/nginx-container/blob/master/imagestreams/nginx-centos.json#L104

this needs to be fixed upstream before we sync samples again in 4.9 or this will be overwritten

at this point I won't block this PR for that

Reply:
That's centos8 image, so its from https://github.com/sclorg/nginx-container/blob/master/imagestreams/nginx-centos.json#L84 (fixed in https://github.com/sclorg/nginx-container/pull/145)

Reply:
ah sorry about that .... lots of permutations first thing in the AM

if you get a sec add https://github.com/sclorg/nginx-container/pull/145 to your nice list of relevant PRs in the description

thanks for the confirmation